statistics.md

```diff
@@ -442,7 +300,7 @@ The following are the `ANALYZE` configurations that support persistence:
 | The `ANALYZE` column type | AnalyzeColumnOption ::= ( 'ALL COLUMNS' \| 'PREDICATE COLUMNS' \| 'COLUMNS' ColumnNameList ) |
 | The `ANALYZE` column | ColumnNameList ::= Identifier ( ',' Identifier )* |
 
-#### Enable ANALYZE configuration persistence
+### Enable ANALYZE configuration persistence
 
```

Comment:
in L327, "Disable ANALYZE configuration persistence" needs to be at the same level ("update it to `### Disable ANALYZE configuration persistence`)